bin/benchmark

```diff
@@ -250,7 +250,7 @@ requests = {
   "textDocument/codeAction" => code_action_params,
   "textDocument/onTypeFormatting" => base_params.merge(position: { line: 1, character: 31 }, ch: "\n"),
   "codeAction/resolve" => base_params.merge(data: { range: range, uri: FILE_URI }),
-  "textDocument/completion" => base_params.merge(position: position),
+  "textDocument/completion" => base_params.merge(position: { line: 1, character: 50 }),
```

Comment:
Line 1 character 50 in the fixture matches the magic comment `frozen_string_literal`. The position here needs to match a forward slash `/` inside of a require path.